docs/standalone.md

```diff
@@ -0,0 +1,36 @@
+# Standalone use
+
+If you already have a cluster (with another provider or whatever), you can still use this Terraform to deploy all the mentioned tools on it, **iff you use ingress-nginx as your ingress controller**. For this, you will need a `kubeconfig` file to access your cluster. (or your credentials, if so, you will have to modify by yourself the `terraform.tf` file).
```

Comment:
```suggestion
If you already have a cluster (with another provider or whatever), you can still use this Terraform to deploy all the mentioned tools on it, **if you use ingress-nginx as your ingress controller**. For this, you will need a `kubeconfig` file to access your cluster. (or your credentials, if so, you will have to modify by yourself the `terraform.tf` file).
```

Reply:
j'ai mis iff pour _if and only if_, c'était pas une typo x)